### PR TITLE
Add more details to "Using Development Version"

### PR DESCRIPTION
**Pull request by vnbrs**
Referring to #439.

Running `rake test` were failing. To overcome that, I had to run `bundle exec rake`, just as it is on the CI.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -232,7 +232,8 @@ If you were previously using Shopify's [activeresource fork](https://github.com/
 Download the source code and run:
 
 ```bash
-rake install
+bundle install --jobs=3 --retry=3
```
Comment (jamiemtdwyer) [inline README.md]: These arguments are only needed when running in the CI environment to make sure that the build doesn't die if there's an issue reaching Rubygems.

`--jobs=3` is used for parallelization and mileage might vary on different workstations. Best to omit it as to not cause confusion.

File: README.md [hunk 2]
```diff
@@ -232,7 +232,8 @@ If you were previously using Shopify's [activeresource fork](https://github.com/
...
+bundle exec rake
```
Comment (jamiemtdwyer) [inline README.md]: For completeness, I think this should be `bundle exec rake test`.

**Comment by vnbrs** [comment]
There you go, @jamiemtdwyer. Thank you for pointing it out!

**Comment by jamiemtdwyer** [comment]
Nice! Thanks a lot @vnbrs.

**Pull request merged**